frontend/views/Home.tsx

```diff
@@ -65,28 +76,65 @@ const styles = StyleSheet.create({
     position: "absolute",
     margin: 16,
     right: 0,
-    bottom: 0,
+    bottom: 64,
+  },
+  view: {
+    height: "100%",
+  },
+  inviteModal: {
+    padding: 20,
+    background: "white",
   },
 });
 
 type Props = {
   navigation: StackNavigationProp<Routes, "Home">;
   route: RouteProp<Routes, "Home">;
+  user: Admin;
   token: string;
 };
 
-function Home(props: Props) {
+type TabProps = {
+  Cultures: { insights: Culture[] };
+  Admins: { insights: Admin };
+};
+
+const Tab = createMaterialTopTabNavigator<TabProps>();
+
+function Home(props: Props): React.ReactElement {
   const [cultures, setCultures] = useState(null);
+  const [users, setUsers] = useState(null);
+  const [editing, setEditing] = useState<boolean>(false);
+  const [err, setErr] = useState<string>("");
+  const [showErr, setShowErr] = useState<boolean>(false);
+  const route = useRoute();
```

Comment:
Is this used? In CultureView it was necessary, but here it doesn't seem like it. Makes useRoute unused as well